index.go

```diff
@@ -242,15 +267,40 @@ func LoadIndexXML(r io.Reader) (*Index, error) {
 	return &index, nil
 }
 
-func (a *App) CurApk() *Apk {
+func (a *App) ApksByVName(vname string) ApkList {
+	var apks ApkList
 	for i := range a.Apks {
-		apk := a.Apks[i]
-		if a.CVCode >= apk.VCode {
-			return &apk
+		if vname == a.Apks[i].VName {
+			apks = append(apks, a.Apks[i])
 		}
 	}
-	if len(a.Apks) > 0 {
-		return &a.Apks[0]
+	return apks
+}
+
+func (a *App) SuggestedApks() ApkList {
+	// No APKs => nothing to suggest
+	if len(a.Apks) == 0 {
+		return nil
+	}
+
+	// First, try to follow CV
+	apks := a.ApksByVName(a.CVName)
```

Comment:
If you publish pre-releases, that's how they will work. That's why we try to keep large minSdk bumps right after a stable release.

You're proposing a completely different approach to finding a recommended version. I still don't understand what do version names have to do with any of this. Either way, I want `fdroidcl` to work in the same way that `fdroidclient` would, otherwise things would get confusing very easily.